src/jade/popup.jade

```diff
@@ -10,8 +10,8 @@ html(class="dollert-popup")
         img(src="logotype.png")
       .dollert-alert-form-label Enter the USD value you are looking for in BRL:
       .dollert-input-buttoned
-        input(data-js="usd-value" placeholder="i.e: 3.70")
-        button.is-primary(data-js="button-save" type="button") Save
+        input(data-js="usd-value" placeholder="e.g: 3.70")
+        button.is-primary(data-js="button-save" type="button").disabled(disabled) Save
```

Comment:
It is better to keep css classes and html attributes grouped:
`button.is-primary.disabled(data-js="button-save" type="button" disabled)`